### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by Traceabl3**
Updated Screeners only.

### CODE REVIEW

File: gamestonk_terminal/screener/presets/rosenwald.ini [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Author of preset: Traceabl3
+# Contact: three shots in the air.
+# Description: the classic rosenwald screen based on some dude i work withs best guess.
```
Comment (DidierRLopes) [inline gamestonk_terminal/screener/presets/rosenwald.ini:3]: The test is failing on the "withs" here, should be "with"

**Review by DidierRLopes — COMMENTED** [review]
Failing on the tests due to codespell. Once that's fixed we can merge this!

You can add to Readme if you want as well, but otherwise it's looking good :)

File: gamestonk_terminal/screener/presets/README.md [hunk 2]
```diff
@@ -2,6 +2,18 @@
 
 * [template](#template)
 * [sexy_year](#sexy_year)
+* [buffett_like](#traceabl3)
```
Comment (DidierRLopes) [inline gamestonk_terminal/screener/presets/README.md:5]: Here should be: 
[buffett_like](#buffett_like)
[cheap_bottom_dividend](#cheap_bottom_dividend)

and so on!

**Review by DidierRLopes — COMMENTED** [review]
Apart from the table of content link on README it's looking good!

File: gamestonk_terminal/screener/presets/rosenwald.ini (hunk 1 above)
Comment (Traceabl3) [reply gamestonk_terminal/screener/presets/rosenwald.ini]: Fixed the preset template, and updated the read me file.

File: gamestonk_terminal/screener/presets/README.md (hunk 2 above)
Comment (Traceabl3) [reply gamestonk_terminal/screener/presets/README.md:5]: no brackets? or all in one set of brackets?

Comment (DidierRLopes) [reply gamestonk_terminal/screener/presets/README.md:5]: Sorry, it formatted as markdown lol.

I meant like this:
```
[buffett_like](#buffett_like)
[cheap_bottom_dividend](#cheap_bottom_dividend)
```

Comment (Traceabl3) [reply gamestonk_terminal/screener/presets/README.md:5]: ah... got it.

Comment (Traceabl3) [reply gamestonk_terminal/screener/presets/README.md]: Done.

**Comment by DidierRLopes** [comment]
Nice work, thanks for this!!

**Review by DidierRLopes — APPROVED** [review]

**Pull request merged**